### PR TITLE
Adds logo/title (Chore)

### PR DESCRIPTION
**Pull request by pierre-1**
# [PT Story](https://www.pivotaltracker.com/story/show/172057805)
## Chore
* Adds logo to client site as per client meeting

## Changes proposed in this pull request:
* Adds Berlingo News logo to Header 
* Comments out hamburger menu

## What I have learned working on this feature:
* Revisiting Semantic UI syntax for image imports and sizing

## Screenshots 
Screenshot 2020-03-31 16 22 40

## Lo-Fi
Screenshot 2020-03-31 16 25 09

### CODE REVIEW

File: src/components/DisplayHeader.jsx [hunk 1]
```diff
@@ -23,9 +24,10 @@ const DisplayHeader = props => {
         >
           {name}
         </Menu.Item>
-        <Menu.Item name="main-header" id="main-header" position="right">
+        <Image src={Logo} size='medium' centered rounded/>
+        {/* <Menu.Item name="main-header" id="main-header" position="right">
           <Icon name="sidebar" size="large"></Icon>
```
Comment (robin-lillqvist) [inline src/components/DisplayHeader.jsx]: Do we need this commented out code?

**Review by robin-lillqvist — COMMENTED** [review]
Left one comment

File: src/components/DisplayHeader.jsx (hunk 1 above)
Comment (pierre-1) [reply src/components/DisplayHeader.jsx]: We have not yet used the functionality of the hamburger menu but were thinking we could implement it when we finalize the styling for the final iteration.

Comment (pierre-1) [reply src/components/DisplayHeader.jsx]: We have decided to remove it and add it as a chore. Thanks!

**Comment by robin-lillqvist** [comment]
LGTM

**Pull request merged**